taprpc/taprootassets.proto

```diff
@@ -1513,6 +1513,15 @@ message AddrReceivesRequest {
 
     // Filter receives by a specific status. Leave empty to get all receives.
     AddrEventStatus filter_status = 2;
+
+    // The number of events to skip.
+    int32 offset = 3;
+
+    // The max number of events returned.
+    int32 limit = 4;
+
+    // The direction of the page. 0 for ascending, 1 for descending.
+    int32 direction = 5;
```

Comment:
It looks like universerpc/universe.proto defines/uses a SortDirection enum for the same purpose. Given that file imports taprpc/taprootassets.proto anyway, I wonder would it be better to move the SortDirection definition to taprootassets.proto and use it in place of 'int32' here?

Reply:
applied by moving `SortDirection` to `tapcommon` and using it in `universe` and `taproot-assets`